packages/discord-request/test/client.test.ts

```diff
@@ -0,0 +1,3 @@
+test("should run", () => {
+	expect(true).toBe(true);
+});
```

Comment:
again, passWithNoTests => true ???

Reply:
Gonna leave this as is since a followup PR removes Jest for Vitest